### ISSUE TITLE
Have file structure mirror the documentation page structure

### ISSUE THREAD

**Issue by lucyleeow** [napari/docs#282]
## 🧰 Task
`docs/guides` contains items that fall under the "In-depth explanations" section of the docs. I find this confusing because we also have a "How-to guides" section of the docs.

Could we rename `docs/guides/` to `docs/explanations/` or something similar?

**Comment by DragaDoncila** [comment]
@lucyleeow yeah i think it would be good for the directory structure to more closely mimic where things end up in the website. I don't know of any good reason not to do this.

**Comment by lucyleeow** [comment]
I've amended this issue to be more general, so it is easier to navigate the docs files.

AFAICT this would involve:

* renaming `docs/guides/` to `docs/explanations/`
* moving `docs/tutorials/fundamentals/` ('Usage' -> 'Getting started') to `docs/getting_started/`

The `docs/further-resources/` file doesn't exactly mirror the docs but I think this is fine to leave.
